app/authorities/sufia/authorities/local/find_works.rb

```diff
@@ -0,0 +1,21 @@
+module Sufia::Authorities
+  class Local::FindWorks < Qa::Authorities::Base
+    attr_reader :q
+
+    def initialize(q)
```

Comment:
Ok. I did a little digging. It is not the actual query as I thought. It appears to be the type? When I check the value of "q" in the init, I get `works`. So should I be calling it "type"?